pymeasure/instruments/hp/hp3478A.py

```diff
@@ -0,0 +1,389 @@
+        hrs = hrs + '     Auto Zero: '+str(status_read[1]>>2&1)+'\r\n'
+        hrs = hrs + '     50 Hz opereration: '+str(status_read[1]>>3&1)+'\r\n'
+        hrs = hrs + '     Front/Rear: '+str(status_read[1]>>4&1)+'\r\n'
+        hrs = hrs + '     CalibrationRAM en: '+str(status_read[1]>>6&1)+'\r\n'
+        hrs = hrs + '     External trigger: '+str(status_read[1]>>7&1)+'\r\n'
+        hrs = hrs + 'Srq mask: '+str(status_read[2])+'\r\n'
+        hrs = hrs + 'Error status: '+str(status_read[3])+'\r\n'
+        hrs = hrs + 'DAC value: '+str(status_read[4])+'\r\n'
+        return hrs
+
+    @property
+    def auto_zero(self):
+        """
+        Checks the autozero settings on the HP3478
+
+        Returns
+        -------
+        autozero : int
+            0 == Autozero off
+            1 == Autozero on
+
+        """
+        status_read=self.status()
+        autozero=status_read[1]>>2&1
+        return autozero
+
+    @auto_zero.setter
+    def auto_zero(self,on_off):
+        """
+        Offers the possibility to enable or disable the Autozero feature
+
+        Parameters
+        ----------
+        on_off : int
+            expects either 0 or 1
+            0 --> disabled autozero
+            1 --> enables autozero.
+
+        Returns
+        -------
+        None.
+
+        """
+        if on_off==0:
+            self.write('Z0')
+        else:
+            if on_off==1:
+                self.write('Z1')
+
+    def active_connectors(self):
+        """
+        checks the status of the front/back switch, selecting either the front
+        or back mearsurement terminals
+
+        Returns
+        -------
+        front_back : int
+            0 == back terminals
+            1 == front terminals
+
+
+        """
+        status_read=self.status()
+        front_back=status_read[1]>>4&1
+        return front_back
+
+    @property
+    def trigger(self):
+        """
+        Checks the triger settings on the HP3478
+
+        Returns
+        -------
+        trigger_stat : str
+            A string value describing the current status of the trigger
+
+        """
+        status_read=self.status()
+        int_trig=status_read[1]&1
+        ext_trig=status_read[1]>>7&1
+        if int_trig==1 and ext_trig==0:
+            trigger_stat='internal'
+        if int_trig==0 and ext_trig==1:
+            trigger_stat='external'
+        if int_trig==0 and ext_trig==0:
+            trigger_stat='hold'
+        return trigger_stat
+
+    @trigger.setter
+    def trigger(self, trigger_mode):
+        """
+        Offers the possibility to set the trigger mode
+
+        Parameters
+        ----------
+        trigger_mode : str
+            allowed values are:
+                 auto, internal, external, single, hold or fast
+
+        Returns
+        -------
+        None.
+
+        """
+        if trigger_mode in self.TRIGGERS.keys():
+            self.write(self.TRIGGERS[trigger_mode])
+        else:
+            print("fix me")
+
```

Comment:
Is it a debug `print` left by mistake? Is it better to raise an exception ? or simply use the code:
`self.write(self.TRIGGERS[trigger_mode])`

Reply:
Thank you very much, i have changed this to an exception